discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
+            "examples": "yes",
+            "eventsAsync": "threaded",
```

Comment:
`coroutine` is poor wording. The asynchronous model is an implementation detail in certain paradigms (e.g. discord.net's TAP). Should just use `'async', 'thread-safe', 'threaded', 'none'`. Same for the apiCallsAsync.